integration-tests/suites/repeated_network_flow.go

```diff
@@ -88,7 +88,28 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	numIter := strconv.Itoa(s.NumIter)
 	sleepBetweenCurlTime := strconv.Itoa(s.SleepBetweenCurlTime)
 	sleepBetweenIterations := strconv.Itoa(s.SleepBetweenIterations)
-	_, err = s.execContainer("nginx-curl", []string{"/usr/bin/schedule-curls.sh", numMetaIter, numIter, sleepBetweenCurlTime, sleepBetweenIterations, serverAddress})
+	_, err = s.execContainerShellScript("nginx-curl", `
+		#!/usr/bin/env sh
+
+		num_meta_iter=$1
+		num_iter=$2
+		sleep_between_curl_time=$3
+		sleep_between_iterations=$4
+		url=$5
+
+		i=0
```

Comment:
Are there other scripts that could be moved into the integration tests in the same way. If so create tickets to do that and add links to them in this PR.

Reply:
I don't see other tests to convert, except ProcessesListening on ports maybe, by piping the c code into the --run option of https://bellard.org/tcc/tcc-doc.html#Option-summary :wink: